SingularityService/src/main/java/com/hubspot/singularity/resources/RequestResource.java

```diff
@@ -125,6 +123,18 @@ private void submitRequest(SingularityRequest request, Optional<SingularityReque
     requestHelper.updateRequest(request, oldRequest, requestState, historyType, JavaUtils.getUserEmail(user), skipHealthchecks, message);
   }
 
+  private void checkForAuthorization(SingularityRequest request, Optional<SingularityRequest> oldRequest, SingularityAuthorizationScope scope) {
```

Comment:
@tpetr this check is in `RequestResource` because passing along the `SingularityUser` to `SingularityValidator` would be confusing